### ISSUE TITLE
Not found

### ISSUE THREAD

**Issue by iatomic1** [synacktraa/ani-skip#14]
Been trying to use the api for a few days now and all the animes I'm testing is not always found, am i the one not doing it right?

**Comment by synacktraa** [comment]
Please show some examples so I can verify.

**Comment by hollowillow** [comment]
I second this issue.

Most of the features seem to be working fine (ie. fetching the MAL ID and other features work) but whenever I attempt to generate the skip script via the --episode argument, it results in a "Skip times not found!" response. 

It's not as if the anime I am trying this out on don't have their skip times present in the API (say One Piece), for the sake of completeness I even tried it on Solo Leveling and Black Clover, using your exact examples as found in the README to no avail either. 

Unfortunately I have no idea what could cause this issue. I am on Arch, using the latest version installed from the AUR package via the yay AUR helper. 

Any help or feedback would be greatly appreciated.

**Comment by jaum20** [comment]
Any update on this?

**Comment by pakosaan** [comment]
the solution is to use --skip-title to make it work with ani-skip. for example:
for one piece, ani-cli uses 1P while ani-skip uses One Piece so we can use the following:
ani-cli --skip-title "One Piece" --skip

**Comment by hollowillow** [comment]
> the solution is to use --skip-title to make it work with ani-skip.

Yep that works. I suppose it does make sense thinking back on it. While ani-skip uses the aniskip API, ani-cli scrapes the allmanga website so they end up with mismatched titles.

**Comment by synacktraa** [comment]
This should be fixed now. Please re-open if it persists!
